lib/apiservers/engine/backends/container.go

```diff
@@ -142,8 +150,8 @@ func (c *Container) ContainerCreate(config types.ContainerCreateConfig) (types.C
 	var err error
 
 	//TODO: validate the config parameters
-	log.Debugf("config.Config = %+v", config.Config)
-
+	log.Printf("Image fetch section - Container Create")
```

Comment:
I added the log.printf(maybe it should be log.Infof) to help break down the massive `ContainerCreate` call. so now you can see the individual sections of the create call you are passing through in the logs. We don't actually need it. But that function is massive and makes several calls to the portlayer. I found it useful to atleast have some logging around which section the call might be failing in.